frontend/src/components/RecruitItem.vue

```diff
@@ -10,8 +10,18 @@
         </div>
       </div>
       <div class="button-wrapper">
-        <button class="enroll-button" @click="onClickAdmission(recruitment.id)">
-          지원하기
+        <button
+          class="enroll-button button"
+          @click="onClickAdmission(recruitment.id)"
+          v-if="this.isRecruiting()"
+        >
```

Comment:
```suggestion
        <button
          class="enroll-button button"
          @click="onClickAdmission(recruitment.id)"
          :disabled="!this.isRecruiting()"
        >
```

`disabled` 속성을 조절하면 버튼 컴포넌트를 하나만 써도될것 같아요

Reply:
그리고 디자인의 통일성을 생각해서 `Button` 컴포넌트를 재사용하는 것도 좋은 방법일 것 같아요 :)